commitlint/README.md

```diff
@@ -52,6 +56,7 @@ jobs:
         uses: extenda/actions/commitlint@v0
         with:
           message: ${{ github.event.pull_request.title }}
+          require-type-and-scope: false
```

Comment:
Don't add this as false to existing examples, instead add a new example that shows how this is added and set to 'true'.